easybuild/easyconfigs/s/Singularity/Singularity-3.5.3-GCC-8.3.0.eb

```
@@ -33,14 +33,12 @@ builddependencies = [
     ('Go', '1.14', '', True),
 ]
 
-start_dir = '%(namelower)s-%(version)s'
-
 configure_cmd = "echo '%(version)s' > VERSION && "
 configure_cmd += "./mconfig -p '%(installdir)s' -b ./buildtree"
 
 prebuildopts = "cd ./buildtree && "
 
-preinstallopts = "cd '%(start_dir)s/buildtree' && "
+preinstallopts = "cd '%(namelower)s-%(version)s/buildtree' && "
```

@robqiao my bad, it should be:
```suggestion
preinstallopts = prebuildopts
```

---

Thx @smoors. It seems that `prebuildopts = "cd ./buildtree && "` is relative to  `%(builddir)%(name)s-%(version)s ` but `preinstallopts` is relative to `%(builddir)s`.

```
            if self.dry_run and os.path.samefile(topdir, self.builddir):
                topdir = os.path.join(self.builddir, '%s-%s' % (self.name, self.version))
                self.log.info("Modified parent directory of start dir in dry run mode to likely path %s", topdir)
                # make sure start_dir subdir exists (cfr. check below)
                mkdir(os.path.join(topdir, start_dir), parents=True)            abs_start_dir = os.path.join(topdir, start_dir)
            if topdir.endswith(start_dir) and not os.path.exists(abs_start_dir):
                self.cfg['start_dir'] = topdir
            else:
                if os.path.exists(abs_start_dir):
                    self.cfg['start_dir'] = abs_start_dir
                else:
                    raise EasyBuildError("Specified start dir %s does not exist", abs_start_dir)
```
I suspect that `preinstallopts = "cd '%(namelower)s-%(version)s/buildtree' && "` would be approperate in this case. Any thoughts?